docs/translations.md

```diff
@@ -1,3 +1,53 @@
+# Managing strings
+
+We use Android's [resource system](https://developer.android.com/guide/topics/resources/localization) to localize
+user-visible strings in our apps.
+
+Our source language is English (American English to be more precise, but simply "English" (en) on Weblate).
+
+Feature modules are expected to store strings exclusively used by them in their `res/values/strings.xml` file.
+There's currently no final place for strings shared between feature modules. For now there's some in
+[:legacy:ui:base](/legacy/ui/base/src/main/res/values/strings.xml).
+
+Translations of source strings happen exclusively in our
+[Weblate project](https://hosted.weblate.org/projects/tb-android/). This means the source language is only modified by
+changes to this repository, i.e. via pull requests. Translations are only updated on Weblate and then merged into this
+repository by the Thunderbird team. This is to avoid overlapping changes in both repositories that will lead to merge
+conflicts.
+
+## Adding a string
+
+Add a new string to the appropriate `res/values/strings.xml` file. Be careful when using Android Studio's translations
+editor. It might modify files containing translations. If so, revert those changes before committing the changes.
+
+## Changing a string
+
+Changing a string should be avoided. Weblate doesn't automatically invalidate translations when a source string is
+changed. This can be worked around by removing the old string and adding a new one. Make sure to only modify the source
```

Comment:
I saw weblate has a plugin to mark source strings as changed, is that something that was considered? Might make changing strings easier, but generally I agree if the meaning is changed then we should have a new key.

I'd suggest slightly more positive wording, i.e. avoiding "workaround" but focusing more on what is the way that developers can make clear to localizers that a string has changed in meaning.

Reply:
I haven't seen those add-ons before. They might be new. 

I saw the user "None" added them to our list of repository-wide installed add-ons a week ago. So maybe that's when they were added to Weblate.

From the documentation it doesn't appear like they do what we want, which is mark all translations of a string as "needs editing" when the string in the source language was changed. 
Instead, the add-on `Flag new source strings as “Needs editing”` marks a source string as "needs editing" on Weblate when it was changed in the repository. Since we don't allow editing source strings on Weblate, there's no point in us using this add-on.
The add-on `Flag new translations as “Needs editing”` will do the same for translated strings. However, we try to avoid changing translations in our repository. And in the rare cases when we do, we don't want them to be marked as "needs editing" on Weblate. So we probably don't want to use this one either.